pkg/logs/client/http/destination.go

```diff
@@ -155,3 +157,19 @@ func buildContentEncoding(endpoint config.Endpoint) ContentEncoding {
 	}
 	return IdentityContentType
 }
+
+// CheckConnectivity check if sending logs through HTTP works
+func CheckConnectivity(endpoint config.Endpoint) config.HTTPConnectivity {
+	log.Info("Checking logs-agent HTTP connectivity...")
+	ctx := client.NewDestinationsContext()
+	ctx.Start()
+	defer ctx.Stop()
+	destination := NewDestination(endpoint, JSONContentType, ctx)
+	err := destination.Send(nil)
```

Comment:
you could probably create a constant emptyContent and add a comment about what's expected, I feel like it's clearer than nil